### PR TITLE
[tools] update tools script

### PR DESCRIPTION
**Pull request by xzj7019**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by xzj7019** [comment]
run buildall

**Review by wm1581066 — APPROVED** [review]
LGTM

**Comment by xzj7019** [comment]
run buildall

File: tools/tpcds-tools/conf/opt/opt_sf1.sql [hunk 1]
```diff
@@ -1 +1,2 @@
 set global runtime_filter_wait_time_ms=1000;
+set global enable_nereids_rules='ELIMINATE_GROUP_BY_KEY';
```
Comment (morrySnow) [inline tools/tpcds-tools/conf/opt/opt_sf1.sql:2]: it is danger to set global here, could we add a post process to set them back to original values?

**Review by morrySnow — DISMISSED** [review]

**Review by keanji-x — APPROVED** [review]

**Comment by xzj7019** [comment]
run buildall

**Review by morrySnow — APPROVED** [review]

**Pull request merged**